_data/simple-icons.json

```
@@ -12070,6 +12070,16 @@
             "hex": "EC008C",
             "source": "https://www.veepee.fr/"
         },
+        {
+            "title": "Vega",
+            "hex": "2450B2",
+            "source": "https://github.com/vega/logos/blob/af32bc29f0c09c8de826aaafb037935fb69e960a/assets/VG_Black.svg",
+            "guidelines": "https://github.com/vega/logos",
+            "license": {
+                "type": "custom",
+                "url": "https://github.com/vega/vega/blob/72b9b3bbf912212e7879b6acaccc84aff969ef1c/LICENSE"
```

- I think we should use the same commit hash as for the source file here.
- I think we should use the license file in the `https://github.com/vega/logos` project.

```suggestion
                "url": "https://github.com/vega/logos/blob/af32bc29f0c09c8de826aaafb037935fb69e960a/LICENSE"
```